benchmark/src/main/resources/META-INF/services/io.gatling.charts.component.ComponentLibrary

```diff
@@ -0,0 +1 @@
+org.keycloak.benchmark.gatling.chart.ComponentLibraryDummy
```

Comment:
Instead of providing dummy implementations can we just add the `io.gatling:gatling-charts` dependency? It's released under the Apache 2.0 license in the main Gatling repository https://github.com/gatling/gatling/tree/main/gatling-charts

Reply:
Despite a naming that might suggest otherwise, it is IMHO as follows: 

* `gatling-charts` provides the base charting functionality, and is still included in the PR
* `gatling-charts` depends on an implementation of the `ComponentLibrary` which is then either the original Highcharts one, or my dummy implementation